src/AWS.Deploy.Recipes/DeploymentBundleDefinitionLocator.cs

```diff
@@ -10,7 +10,7 @@ public class DeploymentBundleDefinitionLocator
         public static string FindDeploymentBundleDefinitionPath()
         {
             var assemblyPath = typeof(DeploymentBundleDefinitionLocator).Assembly.Location;
-            var deploymentBundleDefinitionPath = Path.Combine(Directory.GetParent(assemblyPath).FullName, "DeploymentBundleDefinitions");
+            var deploymentBundleDefinitionPath = Path.Combine(Directory.GetParent(assemblyPath)?.FullName ?? "", "DeploymentBundleDefinitions");
```

Comment:
If by chance `Path.Combine(Directory.GetParent(assemblyPath)?.FullName` fails we will have a bad path for `DeploymentBundleDefinitions` dir. Not failing in this method will propagate the issue in the caller stack.

We must use ! to force unwrap the nullable here. Because we are 100% sure that `Path.Combine(Directory.GetParent(assemblyPath)` will never fail. If it does then we have bigger problem.

Reply:
Fixed